ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialOnboardingLayout.kt

```diff
@@ -0,0 +1,316 @@
+import org.cru.godtools.tutorial.Page
+import org.cru.godtools.tutorial.R
+
+@Composable
+internal fun TutorialOnBoardingWelcomeLayout(
+    nextPage: () -> Unit = {},
+    onTutorialAction: (Int) -> Unit = {},
+) = GodToolsTheme() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            )
+            .padding(horizontal = 32.dp)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.padding(top = 16.dp).minLinesHeight(5, MaterialTheme.typography.bodyMedium))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+
+                /*
+            Spacer for first two elements only
+            Animation will not be in box, will be element of parent column
+
+             */
+
+                Image(
+                    painter = painterResource(org.cru.godtools.base.ui.R.mipmap.ic_launcher_foreground),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .padding(top = 16.dp)
+                        .fillMaxWidth()
+                        .wrapContentSize(Alignment.TopCenter)
+                )
+                Image(
+                    painter = painterResource(R.drawable.ic_logo_title),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .wrapContentSize(Alignment.TopCenter)
+                )
+                Text(
+                    text = stringResource(R.string.tutorial_onboarding_welcome),
+                    style = MaterialTheme.typography.headlineLarge,
+                    textAlign = TextAlign.Center,
+                    color = MaterialTheme.colorScheme.primary,
+                    modifier = Modifier.padding(top = 24.dp)
+                )
+                val bodyText = stringResource(R.string.tutorial_onboarding_welcome_action_video)
+                ClickableText(
+                    text = AnnotatedString(bodyText),
+                    onClick = { onTutorialAction(R.id.action_onboarding_watch_video) },
+                    // style = MaterialTheme.typography.bodyLarge,
+                    style = TextStyle(
+                        color = MaterialTheme.colorScheme.primary,
+
+                    ),
+                    // color = MaterialTheme.colorScheme.primary,
+                    modifier = Modifier.padding(top = 150.dp),
+
+                )
```

Comment:
you actually don't need to use `ClickableText` here, you can use `Text` with a `.clickable()` modifier instead.

The `ClickableText` is useful if you are trying to determine where in a text element the user clicked, but the unfortunate drawback is that it's not part of the material3 widgets, so doesn't use the Material3 theme.